src/Microsoft.Framework.Runtime/DependencyManagement/LockFile.cs

```diff
@@ -9,6 +9,8 @@ namespace Microsoft.Framework.Runtime.DependencyManagement
     public class LockFile
     {
         public bool Islocked { get; set; }
+        public IList<Tuple<string, IEnumerable<string>>> ProjectFileDependencies { get; set; } =
```

Comment:
Create a type for this, this is ugly and it needs to be back ported to NuGet.ProjectModel /cc @anurse